.docker-compose.harvester_ng.yaml

```diff
@@ -0,0 +1,64 @@
+version: '3.2'
```

Comment:
This filename is not named correctly

It should be `.docker-compose-harvester_ng.yaml`
and not `.docker-compose.harvester_ng.yaml`

Reply:
![image](https://user-images.githubusercontent.com/3237309/68609674-bfeef180-0494-11ea-8d7b-5d393aaaa6ec.png)

What's the rule?

Reply:
[Changed](https://github.com/avdata99/ckan-cloud-docker/commit/171c4812e19aef25e6c4c02ea917220d24cae334)